airbyte-config/config-models/src/main/java/io/airbyte/config/Configs.java

```diff
@@ -461,6 +461,11 @@ public interface Configs {
    */
   String getDDDogStatsDPort();
 
+  /**
+   * Set constant tags to be attached to all metrics. Useful for distinguishing between environments.
```

Comment:
Can the javadoc also include an example of how someone would pass a value into this env var?